images/ubuntu/scripts/build/install-python.sh

```
@@ -10,19 +10,24 @@ source $HELPER_SCRIPTS/etc-environment.sh
 source $HELPER_SCRIPTS/os.sh
 
 # Install Python, Python 3, pip, pip3
-apt-get install -y --no-install-recommends python3 python3-dev python3-pip python3-venv
+apt-get install --no-install-recommends python3 python3-dev python3-pip python3-venv
+
+if is_ubuntu24; then
+# Create temporary workaround to allow user to continue using pip
+    sudo cat <<EOF > /etc/pip.conf
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)